dotnet/samples/HomeAutomation/Options/AzureOpenAiOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel.DataAnnotations;
+
+namespace HomeAutomation.Options;
+
+/// <summary>
+/// Loads configuration upon instantiation.
+/// </summary>
+public sealed class AzureOpenAiOptions
```

Comment:
Could we also support OpenAI as part of the sample?

Reply:
Will look into this

Reply:
Added information to address this